config/config_global_test.go

```diff
@@ -72,6 +72,227 @@ global:
 	assert.Equal(t, ":9091", conf.Global.Monitoring[0].Ret.(*PrometheusMonitoring).Listen)	
 }
 
+func TestSyslogLoggingOutletFacility(t *testing.T) {
+	conf := testValidGlobalSection(t, `
+global:
+  logging:
+  - type: syslog
+    level: info
+    format: human
+`)
+	assert.Equal(t, 1, len(*conf.Global.Logging))
+	assert.Equal(t, "local0", (*conf.Global.Logging)[0].Ret.(*SyslogLoggingOutlet).Facility)
+
+	conf = testValidGlobalSection(t, `
+global:
+  logging:
+  - type: syslog
+    level: info
+    format: human
+    facility: kern
```

Comment:
please use go string templating ( https://golang.org/pkg/text/template/ ) or similiar to avoid the repetition here